docs/getting-started/handling-errors.md

```diff
@@ -0,0 +1,326 @@
+
+# Getting started: handling errors
+
+We've covered creating and throwing good errors, now let's talk about handling errors. The way you handle errors, particularly in Express, is important. You can end up writing a lot of code in different parts of your app if you're not careful. More code isn't always bad, but duplicated error handling can lead to your errors not being surfaced to where they're needed.
+
+  * [Bubbling up in Express](#bubbling-up-in-express)
+  * [Don't fear the try/catch block](#dont-fear-the-trycatch-block)
+    * [Error checking](#error-checking)
+    * [Multiple try/catch blocks](#multiple-trycatch-blocks)
+  * [Handling recoverable errors](#handling-recoverable-errors)
+
+
+## Bubbling up in Express
+
+In Express, error handlers can be bound at the Application and Route level. This allows you to write lots of error handling code in one place, and you can rely on your Express error handler to do things such as [log the appropriate error detail](./logging-errors.md), forward them on to a service like Splunk, or render a custom error page.
+
+The following is not a great pattern, as Express has no chance to log the error to a central place, we're missing a lot of information that we should be capturing:
```

Comment:
**nitpick**: comma splice, "we're missing..." etc should be a new sentence

```suggestion
The following is not a great pattern, as Express has no chance to log the error to a central place. We're missing a lot of information that we should be capturing:
```

Reply:
now i think about it more, maybe the sentence should be restructured, because "We're missing a lot of information that we should be capturing" shouldn't be the thing that's introducting the example via the colon

Reply:
desperate for a ☕ here

Reply:
your face is desparate for a coffee

Reply:
@apaleslimghost how about this?

```suggestion
The following is not a great pattern, as Express has no chance to log the error to a central place. This means our logs could be missing information that's essential to debugging this error:
```